src/components/Navigator/NavigatorCard.vue

```diff
@@ -258,11 +258,12 @@ export default {
      * Generates an array of tag labels for filtering.
      * Shows only tags, that have children matches.
      */
-    availableTags: ({ selectedTags, children }) => {
+    availableTags: ({ selectedTags, renderableChildNodesMap }) => {
```

Comment:
`renderableChildNodesMap` represents the full list of nodes displayed in the sidebar, even if they're out of view and not currently being displayed in the recycled DOM element, correct?

I think the answer is yes, and approved if so.

Otherwise, if it's only the subset currently in the DOM, we would run into issues and this is not approved :P

Reply:
100% correct :) Thats any items that can be rendered atm, by expanding their parents.